src/share/poudriere/jail.sh

```diff
@@ -57,8 +57,8 @@ Options:
     -m method     -- When used with -c, overrides the default method for
                      obtaining and building the jail. See poudriere(8) for more
                      details. Can be one of:
-                       allbsd, ftp-archive, ftp, git, http, null, src=PATH, svn,
-                       svn+file, svn+http, svn+https, svn+ssh, tar=PATH
+                       allbsd, ftp-archive, ftp, git, http, null, pkg=URL, ports=PATH,
+		       src=PATH, svn, svn+file, svn+http, svn+https, svn+ssh, tar=PATH,
```

Comment:
Might as well fix the whitespace issue here

Reply:
Its not perfectly tab-aligned, would you prefer I do that or keep the formatting intact?